### DISCUSSION TITLE
could be update the code style  (instance=> instance.member) to (instance>member)

### DISCUSSION THREAD

**Discussion by sgf** [dotnet/csharplang#1006] [General]
could be 

let the (instance=> instance.member) to (instance>member)

that's will make the c# code more clearly

**Comment by yaakov-h** [comment]
Possible duplicate of #91.

**Comment by svick** [comment]
The syntax `instance>member` already means something. Even if the compiler was somehow able to figure out that you meant to use a shorthand lambda and not the greater than operator, it would be incredibly confusing to programmers reading the code.

**Comment by sgf** [comment]
">" just is style, instance@member or other style also is fine.
style is not important，the important is let the code more clearly, like the **yaakov-h** post's https://github.com/dotnet/csharplang/issues/91

**Comment by initram** [comment]
If you are trying to the same thing as the other issue, then why did you create another issue? You could write your comments in the other issue.

**Comment by sgf** [comment]
@initram sorry,i dont know there is another same issue has been at here.i just look for the new C# 8 features not found this improvement,so,im created this issue.
